src/tokens/eth/0xcb03bec536843d338ac138205a6689d4cdc11046.json

```diff
@@ -7,7 +7,7 @@
   "decimals": 18,
   "website": "http://abptcrypto.com",
   "logo": {
-    "src": "https://res.cloudinary.com/cloudimgstorage/image/upload/v1557398000/MBC-LOGO-256X256.png",
+    "src": "https://res.cloudinary.com/cloudimgstorage/image/upload/v1560589154/ABPT-LOGO.png",
```

Comment:
This Logo Path is ok so please add this one: https://res.cloudinary.com/cloudimgstorage/image/upload/v1560589154/ABPT-LOGO.png